main/EDPftvarcon.F90

```diff
@@ -2114,12 +2155,83 @@ subroutine FatesCheckParams(is_master, parteh_mode)
            call endrun(msg=errMsg(sourcefile, __LINE__))
            
         end if
+        
+        ! Check to see if cohort tracking is on if age-senescence is on
+        !----------------------------------------------------------------------------------
+        
+        if ( ( EDPftvarcon_inst%mort_ip_age_senescence(ipft) < check_initialized ) .and. &
+             ( cohort_age_tracking .neqv. .TRUE. ) ) then
+
+           write(fates_log(),*) 'ip_age = ', EDPftvarcon_inst%mort_ip_age_senescence(ipft)
+           write(fates_log(),*) 'cohort_age_tracking = ', cohort_age_tracking
+
+           write(fates_log(),*) 'Age-dependent mortality cannot be on'
+           write(fates_log(),*) 'if cohort age tracking is off'
+           write(fates_log(),*) 'Set cohort_age_tracking to 1'
+           write(fates_log(),*) 'To turn on cohort age tracking or '
+           write(fates_log(),*) 'Set mort_ip_age_senescence to _ '
+           write(fates_log(),*) 'to turn off age-dependent mortality '
+           write(fates_log(),*) 'Aborting'
+           call endrun(msg=errMsg(sourcefile, __LINE__))
+        end if
+
+
+        ! Check that parameter ranges for age-dependent mortality make sense   
+        !-----------------------------------------------------------------------------------    
+        if ( ( EDPftvarcon_inst%mort_ip_age_senescence(ipft) < check_initialized ) .and. &
+             (  EDPftvarcon_inst%mort_r_age_senescence(ipft) > check_initialized ) ) then
+
+           write(fates_log(),*) 'Age-dependent mortality is on'
+           write(fates_log(),*) 'Please also set mort_r_age_senescence'
+           write(fates_log(),*) 'Sensible values are between 0.03-0.06'
```

Comment:
I like the 'sensible values' guidance!!

Reply:
That was @rgknox 's idea :) . They rate parameter is not very intuitive so hopefully this stops people wasting time experimenting with values that give unrealistic results.